chrissi/index.js

```diff
@@ -0,0 +1,60 @@
+//0
+
+let snackServings = 4;
+let guests = 16;
+
+console.log(snackServings, guests);
+
+// 1 ADD
+console.log(snackServings + guests);
+
+// 2 SUB
+console.log(snackServings - guests);
+
+// 3 MULT
+console.log(snackServings * guests);
+
+// 4 DIV
+console.log(snackServings / guests);
+console.log(guests / snackServings);
+
+// 5
+let drinks = 10;
+
+// (snackServings * guests) = 64
+
+console.log(64 / drinks);
```

Comment:
You have to get rid of 64 and also console.log(64 / drinks). Just make a variable  snackServings * guests instead and show the result in console and the code should be like this =>
const resultOne = (snackServings * guests) / drinks;
console.log(resultOne)

Reply:
done